frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -532,3 +547,19 @@ function ViewTitleHeaderRightSide(props) {
 }
 
 ViewTitleHeader.propTypes = viewTitleHeaderPropTypes;
+
+function getMissingRequiredTemplateTagsTooltip(requiredTemplateTags = []) {
+  if (!requiredTemplateTags.length) {
+    return "";
+  }
+
+  const names = requiredTemplateTags
+    .map(t => `"${t["display-name"] ?? t.name}"`)
```

Comment:
It took me a moment to realize the `t` is not from `ttag`.
Please use meaningful names :pray: